src/main/java/org/prebid/server/proto/openrtb/ext/request/interactiveoffers/ExtImpInteractiveoffers.java

```diff
@@ -10,5 +10,5 @@
 @Value
 public class ExtImpInteractiveoffers {
 
-    Integer pubid;
+    String partnerId;
```

Comment:
if you need this property in json to be "partnerId" - you need to add @JsonProperty here with this value, otherwise it transforms to "partner_id"

Reply:
changed